### DISCUSSION TITLE
Bug Details while downloading

### DISCUSSION THREAD

**Discussion by satyamgipu** [aclap-dev/video-downloadhelper#631] [Help and Q&A]
### Your question

Downloading failed when try to download HLS video, if it is not supported?

### Addon & CoApp version and details

{
  "version": "9.0.1.18",
  "target": "google",
  "channel": "stable",
  "page_url": "https://courses.growdataskills.com/s/courses/643d0d95e4b0f499814b8e4f/take",
  "error": {
    "details": "SideDownload error: : [hls @ 0000017c26c9a700] Error when loading first segment 'https://dnqe0v482iyb1.cloudfront.net/spees/w/o/6437cee9e4b0fd108b394327/v/64a5a715e4b0185f6cf7c650/u/6479b5ebe4b06efce6244d89/t/235b2c9aa839e88e9a3cbf7b91f9e086/p/assets/videos/6437cee9e4b0fd108b394327/2023/07/05/64a5a715e4b0185f6cf7c650/hls_1M_000.ts'\r\n[in#0 @ 0000017c26c9a580] Error opening input: Invalid data found when processing input\r\nError opening input file https://dnqe0v482iyb1.cloudfront.net/spees/w/o/6437cee9e4b0fd108b394327/v/64a5a715e4b0185f6cf7c650/u/6479b5ebe4b06efce6244d89/t/235b2c9aa839e88e9a3cbf7b91f9e086/p/assets/videos/6437cee9e4b0fd108b394327/2023/07/05/64a5a715e4b0185f6cf7c650/hls_1M_.m3u8.\r\nError opening input files: Invalid data found when processing input\r\n",
    "downloadable_id": "downloadable_2678362726003289",
    "error": "coapp_failure",
    "id": "coapp_failure_36f76607-6faa-415a-904a-77c481f9825c"
  }
}

**Comment by GrampaWildWilly** [comment]
You'll want to read #611.  Your case is a duplicate of that one.
